src/pages/iou/ReceiptSelector/index.js

```diff
@@ -108,86 +124,193 @@ function ReceiptSelector(props) {
+                    />
+                </PressableWithFeedback>
+                <PressableWithFeedback
+                    accessibilityRole={CONST.ACCESSIBILITY_ROLE.BUTTON}
+                    accessibilityLabel={translate('receipt.flash')}
+                    style={[styles.alignItemsEnd, !isTorchAvailable && styles.opacity0]}
+                    onPress={toggleFlashlight}
+                    disabled={!isTorchAvailable}
+                >
+                    <Icon
+                        height={32}
+                        width={32}
+                        src={Expensicons.Bolt}
+                        fill={isFlashLightOn ? themeColors.iconHovered : themeColors.textSupporting}
+                    />
+                </PressableWithFeedback>
+            </View>
+        </>
+    );
+
+    const desktopUploadView = () => (
+        <>
+            <View onLayout={({nativeEvent}) => setReceiptImageTopPosition(PixelRatio.roundToNearestPixel(nativeEvent.layout.top))}>
+                <ReceiptUpload
+                    width={CONST.RECEIPT.ICON_SIZE}
+                    height={CONST.RECEIPT.ICON_SIZE}
+                />
+            </View>
+
+            <View
+                style={styles.receiptViewTextContainer}
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                {...panResponder.panHandlers}
+            >
+                <Text style={[styles.textReceiptUpload]}>{translate('receipt.upload')}</Text>
+                <Text style={[styles.subTextReceiptUpload]}>
+                    {isSmallScreenWidth ? translate('receipt.chooseReceipt') : translate('receipt.dragReceiptBeforeEmail')}
+                    <CopyTextToClipboard
+                        text={CONST.EMAIL.RECEIPTS}
+                        textStyles={[styles.textBlue]}
+                    />
+                    {isSmallScreenWidth ? null : translate('receipt.dragReceiptAfterEmail')}
+                </Text>
+            </View>
+
+            <AttachmentPicker>
+                {({openPicker}) => (
+                    <Button
+                        medium
+                        success
+                        text={translate('receipt.chooseFile')}
+                        accessibilityLabel={translate('receipt.chooseFile')}
+                        style={[styles.p9]}
+                        onPress={() => {
+                            openPicker({
+                                onPicked: (file) => {
+                                    setReceiptAndNavigate(file, iou, report);
+                                },
+                            });
+                        }}
+                    />
+                )}
+            </AttachmentPicker>
+        </>
+    );
+
+    return (
+        <View style={[styles.flex1, !Browser.isMobile() && styles.uploadReceiptView(isSmallScreenWidth)]}>
+            {!isDraggingOver && (Browser.isMobile() ? mobileCameraView() : desktopUploadView())}
```

Comment:
I know you didn't add the `Browser.isMobile` function but I am very surprised to see it included in our codebase..

Reply:
Do you mean we should depend on the screen size breaking points instead of classifying browsers as mobile / desktop?

Reply:
No - I was just surprised to see this function exist and it brought up an internal discussion, no need to change code.